pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -175,6 +188,39 @@ func RunCreate(cmd *cobra.Command, args []string) {
 	}
 }
 
+// RunRegenerate runs a CLI command
+func RunRegenerate(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+
+	if len(args) != 1 || args[0] == "" {
+		log.Fatalf(`❌ Missing expected arguments: <noobaa-account-name> %s`, cmd.UsageString())
+	}
+
+	var decision string
+	log.Printf("You are about to regenerate an account's security credentials.")
+	log.Printf("This will invalidate all connections between S3 clients and NooBaa which are connected using the current credentials.")
+	log.Printf("are you sure? y/n")
+	
+	for {
+		fmt.Scanln(&decision)
+		if decision == "y" {
+			break
+		} else if decision == "n" {
+			return 
+		}
+	}
+
+	name := args[0]
```

Comment:
Missed this on the previous review but I think we should check that this account exists, especially when first thing you do is to check if the secret exists and then the user will get the wrong error - no secret instead of no account with this name. also will probably do the check before of asking the "are you sure". something like this:
```suggestion
        o := util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_noobaaaccount_cr_yaml)
        noobaaAccount := o.(*nbv1.NooBaaAccount)
	noobaaAccount.Name = args[0]
	noobaaAccount.Namespace = options.Namespace

	if !util.KubeCheck(noobaaAccount) {
		log.Fatalf(`❌ Could not get NooBaaAccount %q in namespace %q`,
		noobaaAccount.Name, noobaaAccount.Namespace)
	}
```

Reply:
I am not sure, as the account can exist but not an object, for example, admin@noobaa.io account has no `noobaaaccount` object.
With that said, if there is no such object then there will be no secret which means this will not work on defaults accounts. 
@nimrod-becker Do we want it enabled on default accounts?

Reply:
Why? We want to be able to change for accounts which were created via UI/RPC and not only CRD, for those there would be no object

Reply:
ok, so first we need to handle that - cause no secret will be found (unless it's admin and then we need to do some special handling). and second we can read the account first using the API and then warn him in case the user doesn't exist - before doing anything else.

Reply:
Ok, added the regenerate to none CRD based accounts.

We will check if there is a CRD then we will go to the proper flow.

a gap remains in other flows (list, delete) that we are handling only CRD based accounts.